showcase/app/components/shw/divider.gts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { concat } from '@ember/helper';
+
+export interface ShwDividerSignature {
+  Blocks: {
+    default: [];
+  };
+  Args: {
+    level?: 2;
+  };
+  Element: HTMLHRElement;
```

Comment:
[nitpick] better to follow the ["standard" order in the signature:](https://hashicorp.atlassian.net/wiki/spaces/HDS/pages/3245932580/Using+Typescript#Signature) 
```suggestion
  Args: {
    level?: 2;
  };
  Blocks: {
    default: [];
  };
  Element: HTMLHRElement;
```

Reply:
I removed the Blocks again from what you said below